### DISCUSSION TITLE
How to migrate from wagtailtrans to simple_translation

### DISCUSSION THREAD

**Discussion by jkevingutierrez** [wagtail/wagtail#11380] [Ideas]
It would be great to have documentation about how to move from [wagtailtrans](https://github.com/wagtail/wagtailtrans) to simple_translation

### Pertinent section of the Wagtail docs 

https://docs.wagtail.io/en/latest/reference/contrib/simple_translation.html#simple-translation

or 

https://docs.wagtail.io/en/stable/advanced_topics/i18n.html#alternative-internationalisation-plugins

### Details

Now that wagtail have multi-language support with simple_translation it would be good to have a migration guide from wagtailtrans

**Comment by allcaps** [comment]
Great suggestion! Although, I'm not sure if Wagtail should document migration paths for third-party packages. 🤔 .

A Wagtail Page has a [TranslatableMixin](https://github.com/wagtail/wagtail/blob/main/wagtail/core/models.py#L720) that adds the [`translation_key` and `locale` fields](https://github.com/wagtail/wagtail/blob/0dfc342d6636c3c9a220f1fabea23955c7242299/wagtail/core/models.py#L406-L407). Simple translation is _only_ a user interface. Activating Wagtail Simple Translation doesn't change data structure. Upgrading to a recent Wagtail does.

This guide is about migrating form WagtailTrans to Wagtail Localize:
https://www.cnc.io/en/blog/wagtailtrans-to-wagtail-localize-migration

Going to Wagtail Simple Translation is similar: Just don't install Wagtail Localize but activate Wagtail Simple Translation.

Additional details in: https://github.com/wagtail/wagtailtrans/issues/208#issuecomment-757515962

**Comment by thibaudcolas** [comment]
Going to agree with @allcaps on this, it’s not for Wagtail docs to document migration from specific versions of specific packages. We could however link to existing migration guides if it helps. Converting this to Discussions as it’s still a pretty cool idea even if not something for Wagtail docs.
